web/client/examples/viewer/components/FullScreen.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var {ButtonGroup} = require('react-bootstrap');
+var {Button, Glyphicon} = require('react-bootstrap');
+
+var full = React.createClass({
+    propTypes: {
+        activeKey: React.PropTypes.string
+    },
+    getInitialState() {
+        return {
+            active_full_button: false,
+            active_resize_button: true,
+            activeKey: this.props.activeKey
+        };
+    },
+    render() {
+        return (
+                <ButtonGroup id="screen-switcher" type="select" bsSize="small">
```

Comment:
Instead of using 2 buttons, use only one with the icon and the text that changes with the fullScreen prop .